web/README.md

```diff
@@ -27,13 +27,68 @@ To fix the code formatting and check for common errors, run:
 yarn run lint
 ```
 
+### End-to-End (E2E) Testing
```

Comment:
I think this might be more appropriate to put in a new `README.md` at https://github.com/dandi/dandi-archive/tree/master/e2e, as presumably that is where a new dev would start if they are interested in running the e2e tests.